server/src/projects/project-applicants/project-applicant.controller.ts

```diff
@@ -0,0 +1,85 @@
+import { Controller, Patch, Body, Param, Post, UseInterceptors, UseGuards, UsePipes, Delete, HttpException, HttpStatus } from '@nestjs/common';
```

Comment:
Looks like `Patch` and `Param` are imported but not used -- we should delete them